.github/workflows/build-rule-book.yaml

```diff
@@ -16,6 +16,12 @@ on:
 jobs:
   build_rule_book:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
is this to catch all errors in all languages just in case?

Reply:
If set to true (the default), when a single job fails, all the other jobs are cancelled. We don't want that. We want all of them to complete.